bpf/init.sh

```diff
@@ -184,6 +185,10 @@ function encap_fail()
 	# If the host does not have an IPv6 address assigned, assign our generated host
 	# IP to make the host accessible to endpoints
 	if [ "$IP6_HOST" != "<nil>" ]; then
+                echo ">>>>>>>>>> $STALE_IP6_HOST"
+                if [ "$STALE_IP6_HOST" != "<nil>" ]; then
+                    ip -6 addr del $STALE_IP6_HOST dev $HOST_DEV1
+                fi
```

Comment:
@brb Martynas, we can either delete the stale ipv6 here at init.sh or (see the other comment below)